doc/changes/changes_1.0.3.md

```diff
@@ -0,0 +1,24 @@
+# cloudwatch-adapter 1.0.3, released 2022-06-24
+
+Code name: Dependency Updates
+
+## Summary
+
+In this release we updated dependencies and by that fixed the following security vulnerabilities: `CVE-2022-24823`, `sonatype-2020-0026`, `CVE-2021-22569`, `CVE-2020-36518`.
```

Comment:
Github has an auto-link feature for CVEs. I doubt this works if they are enclosed in backticks. Please check and remove the backticks if so.

Reply:
Maybe use a bullet list for readability.